packages/components/javascript/AGENTS.md

```diff
@@ -0,0 +1,182 @@
+# what3words — Autosuggest component (no map) Agent Guide (CDN‑only)
```

Comment:
Both of these are using the older version of this file. I created a slightly newer one in the ticket which includes a disclaimer to any humans who find this. Can you update these files with that version please?